Added Contact form

ahmadsarfrazx opened the pull request:
- Added h1 tag in the Contact form
- Added description in the Contact form
- Added the contact form
- Added styles to it, to make it look like a Figma template
- Added a way to receive the Forms submitted by others
- Removed the icon bar from the bottom of the contact page
- Increase font-weight to 400 to match the Figma template
- Added background image in the image folder

ahmadsarfrazx committed:
Added contact us page in HTML file

asifaminisonline committed:
Added some CSS to contact form

ahmadsarfrazx committed:
Added bar and pictures to HTML or CSS files

levy002 commented on style.css:
```diff
@@ -351,3 +351,97 @@ h2 {
   gap: 15px;
 }
+
+.contact-form {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-image: url(assets/images/background2.png);
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: 70%;
+  background-position-x: right;
+  background-position-y: 20px;
+  border-radius: 90px 0 0 0;
+  padding-top: 103px;
+}
```
- [ ] Kindly make sure the background image `assets/images/background2.png` you are targeting is stored in your images folder.

    - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/199469311-1b34e5b4-73fa-4bfa-93a8-8d92c09cfcfe.png)

  - ### Figma design
 
   ![image](https://user-images.githubusercontent.com/87197412/199469710-a44fdf89-92f5-4827-ad26-39036af17273.png)

levy002 commented on style.css:
```diff
@@ -351,3 +351,97 @@ h2 {
+  margin-bottom: 20px;
+}
+
+.p-contact {
+  font-style: normal;
+  font-weight: 300;
+  font-size: 18px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  padding: 24px;
+}
```
- [ ] Kindy make sure to use to correct typography on this paragraph in order to meet the provided Figma design. :wink: 

   - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/199470128-0182606a-c0e3-4b95-9935-605eb7286eea.png)

   - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/199470213-ca3d726c-d24a-4376-923e-25ac3268af99.png)

levy002 commented on index.html:
```diff
@@ -252,6 +252,29 @@ <h2 class="heading-2">About <br />Myself</h2>
+          <small id="message-success" class="hidden">From Submitted</small>
+          <input type="submit" class="btncontact" value="Get It Touch" />
+        </form>
+        <div class="contact-bar">
+          <img class="end-line" src="assets/images/Shape.png" alt=" end line icon" />
+        </div>
```
- [ ] KIndly remove this line image even if it is visible on the Figma but its role is to indicate that the design is for mobile versions. Kindly remove it.

    - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/199470632-a0c85f81-4ef3-405c-90f2-95f2a69782c4.png)

levy002 reviewed (changes requested):
## Hello @ahmadsarfrazx   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ahmadsarfrazx committed:
Did some required chnages in html and css files

ahmadsarfrazx commented:
- Removed the icon bar from the bottom of the contact page
- Increase font-weight to 400 to match the Figma template
- Added background image in the image folder

ha-manel commented on style.css:
```diff
@@ -351,3 +352,89 @@ h2 {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-image: url(assets/images/background2.png);
```
- [ ] Kindly add the background image like the one used in the template you chose, please use the correct path to the image (background2.png doesn't exist) and you have a background image from another template in you assets folder 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/199596587-38761f71-568e-4cd6-8a60-02e1b9780b4a.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/199596621-b260eed7-904b-4b5c-86c0-678eb9dad868.png)

ha-manel reviewed (changes requested):
# STATUS: REQUIRED CHANGES ♻️

## Hi @ahmadsarfrazx 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  

<img src="https://media3.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>

## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ahmadsarfrazx committed:
Added background in contact form

ahmadsarfrazx commented:
I have added background images in the "main" and "contact-form" branches. Background image is perfectly working in my live server.

Alfredbis29 reviewed (approved):
# STATUS: APPROVED  
 ## Hi @ahmadsarfrazx ,
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your repo is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:

![](https://camo.githubusercontent.com/54a5d803a138b5a1a1242a607e4c1192c852cab8c591c1ddcaa33d06b51560f6/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f686c516559335062684833692f67697068792e676966)
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
